contrib/storage-jdbc/src/test/java/org/apache/drill/exec/store/jdbc/TestJdbcPluginWithH2IT.java

```diff
@@ -53,6 +54,10 @@
   @BeforeClass
   public static void init() throws Exception {
     startCluster(ClusterFixture.builder(dirTestWatcher));
+    // Force timezone to UTC for these tests.
```

Comment:
I recommend that :
```java

  private static TimeZone defaultTimeZone;

  @BeforeClass
  public static void setUp() {
    defaultTimeZone = TimeZone.getDefault();
    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
  }

  @AfterClass
  public static void cleanUp() {
    TimeZone.setDefault(defaultTimeZone);
  }
```

Reply:
Good call.  I wonder if it might actually make sense to include that in the actual test architecture as there have been a lot of issues with unit tests and timezones.  What do you think?

Reply:
Actually, I really don't recommend setup the timezone in UT. However, I changed my mind when refactor the image plugin. Because the library of image not supported to set timezone by API (Get the timezone of host automatically), the unit tests required to compare the value and only provide the constant value by `testBuilder().jsonBaselineFile()`. So, I have to set the timezone in the unit tests. I think there's no problem if we have to  pass the tests.

Reply:
Fixed.